azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/style/ChatCompositeColors.kt

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.chat.presentation.style
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+
+@Immutable
+data class ChatCompositeColors(
```

Comment:
We should put the default values in the constructor, so we don't need to redefine in both the staticCompositionOf and the Theme, unless customizing.

However will approve PR, we can refactor/improve in a another pr.